onyo/lib/commands.py

```diff
@@ -604,6 +607,8 @@ def onyo_mkdir(inventory: Inventory,
         Paths to directories which to create.
     message
         An optional string to overwrite Onyo's default commit message.
```

Comment:
This should be updated (and others). I only point it out again because it doesn't mention "custom". Rather, all `message` parameters should be checked/unified.